R/static_cfr.R

```diff
@@ -0,0 +1,118 @@
+#' Static case fatality rate
+#'
+#' @description Calculates the CFR at a single point in time using in the case
+#' and death time series supplied. Uses all of the time points supplied by the
+#' user and returns an estimate for a single point in time.
+#'
+#' @param df_in A data.frame containing the outbreak data. A daily time series
+#' with dates or some other absolute indicator of time (e.g. epiday/epiweek) and
+#' the numbers of new cases and new deaths at each time point
+#'
+#' @param correct_for_delays A boolean flag indicating whether the user wishes
+#' to correct for the delay between case detection and death. FALSE corresponds
+#' to a naive CFR being calculated, TRUE corresponds to the user calculating a
+#' corrected CFR
+#'
+#' @param delay_pmf The delay distribution used, in the form of a probability
+#' mass function parameterised by time. I.e. f(t) which gives the probability a
+#' case has a known outcomes (i.e. death) at time t, parameterised with
+#' disease-specific parameters before it is supplied here. A typical example
+#' would be a symptom onset to death delay distribution
+#' @param poisson_threshold The case count above which to use Poisson
+#' approximation. Set to 200 by default.
```

Comment:
It would be good to use roxygen2's `@inheritParams` feature to avoid copy/pasting the docs for shared arguments across functions

Reply:
Param for `estimate_ccfr` now inherited from `static_cf`, see https://github.com/epiverse-trace/datadelay/commit/74af9360e3c410d63b971c81235547872df95f53